layers/+emacs/emacs-info/README.org

```diff
@@ -0,0 +1,10 @@
+#+TITLE: Emasc Info Imrprovements
+
+#+TAGS: emacs|layer
+
+* Description
+  This layer improves navigation for Emacs Info mode
+** Features:
+   - TODO: list features
+* Key bindings
+  - TODO: list keys
```

Comment:
You should document the key bindings in your layer please refer to the `helm-layer` for a good example of how to document transient key bindings.